### ISSUE TITLE
Tool(s) mail does not work

### ISSUE THREAD

**Issue by stevenbaert** [open-webui/open-webui#4895]
# Bug Report

## Installation Method

Docker 

## Environment

- **Open WebUI Version:** [v0.3.15]

- **Operating System:** [Mac Frontend Docker on Linux]
- **Browser (if applicable):** [Edge]

**Confirmation:**

- [X ] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
Not relevant in this case. 
- [ ] I have included the Docker container logs.
Not relevant in this case since nothing appearing related to this issue. 
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Send mail (and other tools). 

## Actual Behavior:

Nothing

## Description

**Bug Summary:**
Tool mil is not used if I ask it to mail. 

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

In short: when selecting weather tool in chat and weather is asked, you can clearly see the toll is called and you get the weather.

When I use the mail tool from community
https://openwebui.com/t/cvaz1306/email_tools

and select it in chat, then ask to mail it does not say the tool is called.
Also it says sometimes it mails, but it did not and vice versa. However never it shows the tool was called as within the weather tool.

I tried to create other tools myself  and it seems also there it does not indicate at all it understands it should call the tool.

Tried with GPT-4o.

Am I missing something here?
Special way to call tool or debug the calling? Little to find about it on forums, docs or logs.

Thanks for your feedback!
S.
